frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -1201,17 +1146,20 @@ export default ({
       const fromIsJoined = from.isJoined
 
       const initAfterSynced = (toChatRoomId) => {
-        if (toChatRoomId !== this.summary.chatRoomID || this.ephemeral.messagesInitiated) return
-        return this.setInitMessages()
+        if (toChatRoomId !== this.summary.chatRoomID || // If the user has switched to another chatroom during syncing, no need to process the chatroom that has been swithed away.
+          this.ephemeral.messagesInitiated) return
+        this.processSwitchQueue()
```

Comment:
Missing `catch`

Reply:
FYI, you can avoid adding `catch` to all these calls to `processSwitchQueue()` by surrounding the entire contents of the `processSwitchQueue` implementation in a `try/catch` block.